client/.gitignore

```diff
@@ -35,6 +35,7 @@ npm-debug.log*
 /electron/build/
 /electron/dist/
 /electron/out/
+/electron/electron/
```

Comment:
Curious about why this line was added, can you explain it to me?

Reply:
It's a result of removing capacitor-electron from client/.  The commands were previously ran from `client/`, now we do `cd electron` before. When doing an electron:open, it places some stuff inside `electron/electron`. I tried playing with CAPACITOR_ROOT_DIR (https://github.com/Scille/capacitor-electron/blob/19e2c79f354c385c98a1dd1507ea248f6c91e9f2/src/cli-scripts/open.ts#L8) with no success. And since it doesn't seem to change anything...